README.md

```diff
@@ -29,6 +29,11 @@ source name_of_your_virtual_env/bin/activate
 ```
 export PYTHONPATH="${PYTHONPATH:+/Path/to/this/repo/pytest-aug2019}"
 ```
+* Set the config environment variable:
+
+```
+export APP_SETTINGS=config.py
+```
 * Install required libs in your virtual environment:
 ```
 pip install -r requirements.txt

```

Comment:
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'

Reply:
Fixed.